workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_home_resources.scss

```diff
@@ -0,0 +1,101 @@
+.home-resources {
+	display: grid;
+	gap: 48px;
+	grid-template-columns: repeat(auto-fit, 184px);
+	grid-template-rows: 150px;
+	max-width: 948px;
+	padding: 0 1.3125rem;
+
+	@media (max-width: 1200px) {
+		gap: 24px;
+		grid-template-columns: repeat(2, 184px);
+	}
+
+	@media (max-width: 576px) {
+		grid-template-columns: repeat(1, 184px);
+	}
+
+	.resource {
+		background-color: #fff;
+		border: 1px solid rgba(9, 16, 24, 0.12);
+		border-radius: 0.25rem;
+		box-shadow: 0 6px 15px -6px rgb(9 16 29 / 6%);
+		min-width: 184px;
+		padding: 1.8125rem 1rem;
+		text-decoration: none;
+
+		&:not(.card-disabled):hover {
```

Comment:
@jotchon  what is "card-disabled" for? I don't see it set anywhere

Reply:
That's from an old css selector.  I'll go ahead and remove that.

Reply:
@jotchon  I would like to try to avoid writing CSS as much as possible, so please try to be careful about copying in from other sources.